crates/gpui/README.md

```diff
@@ -5,7 +5,9 @@ for Rust, designed to support a wide variety of applications.
 
 ## Getting Started
 
-GPUI is still in active development as we work on the Zed code editor and isn't yet on crates.io. You'll also need to use the latest version of stable rust and be on macOS. Add the following to your Cargo.toml:
+GPUI is still in active development as we work on the Zed code editor and isn't yet on crates.io. You'll also need to use the latest version of stable rust and be on macOS. If `xcrun metal` is _unable to find utility "metal"_, install `xcode` and run `sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer`.
```

Comment:
Can we put this down below, as it is essentially a troubleshooting step?

I don’t think it reads well in this paragraph.